src/content/news/2025-06-02-maplibre-newsletter-june-2025/index.mdx

```diff
@@ -0,0 +1,244 @@
+Read on to catch up on updates from last month and a glimpse of what's ahead.
+
+## 📱 MapLibre Native
+
+**Plugin Layers are coming:** A major step towards customizability, plugin layers will soon allow developers to register custom layers with their own style spec extensions. The [design proposal](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2025-05-08-plugin-layers.md) was merged this month, and the [implementation PR](https://github.com/maplibre/maplibre-native/pull/3430) led by _Malcolm Toon of HudHud_, is nearing completion. HudHud also released their [navigation app](https://apps.apple.com/us/app/hudhud-maps/id6477774492) this month, which showcases MapLibre Native in production.
+
+Note: The first version of plugin layer is _**iOS only**_ and comes with a few limitations.
+
+- Custom layers do not have access to built-in data sources.
+- Developers must interact with low-level graphics API (e.g. Metal) directly.
+- Platform specific rendering like Drawables cannot be used.
+- Lastly, plugin layers cannot participate in label placement.
+
+<br></br>
+Despite these constraints, this marks a solid functional foundation. Further
+capabilities can be added incrementally based on community needs. Plugin
+extendability is something that was often discussed the last few years, and it
+is great to see a functional and pragmatic implementation becoming reality.
+Malcolm also shared an impressive GLTF model loading plugin as a proof of
+concept.
+
+{" "}
+
+{" "}
+
+<div
+  style={{
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: "1rem",
+    marginBottom: "1rem",
+  }}
+>
+  <img
+    src="https://github.com/user-attachments/assets/a6789d4e-7734-4f57-b41b-0c15b55e79d5"
+    alt="GLTF plugin demo"
+    style={{
+      width: "40%",
+      height: "auto",
+      maxHeight: "550px",
+      objectFit: "contain",
+    }}
+  />
+  <div
+    style={{
+      fontStyle: "italic",
+      fontSize: "0.95rem",
+      textAlign: "center",
+      marginTop: "0.5rem",
+      color: "#444",
+    }}
+  >
+    GLTF model plugin, rendering the Arc de Triomphe in 3D. Satellite layer
+    courtesy of MapTiler.
+  </div>
+</div>
+
+**Action Journal for Debugging:** Thanks to _Adrian Cojocaru_, a new [action journal](https://github.com/maplibre/maplibre-native/commit/08eb760acdf88692139cc9d777ed95074662387e) was merged this month. This is a persistent log of actions the library took, sort of like a black box in a plane, which should help with investigating crashes that are hard to reproduce. Check out the documentation for [iOS](https://maplibre.org/maplibre-native/ios/latest/documentation/maplibre-native-for-ios/actionjournalexample) or [Android](https://maplibre.org/maplibre-native/android/examples/observability/action-journal/) for more information.
```

Comment:
Why do you italicize names of people? This is only used for names of literature, or foreign words, not personal names or names of companies.

Could you change it here and earlier?

Reply:
I wanted to highlight the contributors with a different styling. I am not a native speaker and do not know the rules of the language very well. Retaining as indicated in the original.